docs/modules/ROOT/pages/explanations/decisions/local-objectstorage.adoc

```diff
@@ -0,0 +1,100 @@
+= VSHN Managed Object Storage
+
+== Problem
+
+Having access to S3 compatible storage is required for VSHN managed AppCat services, as well as the APPUiO Managed cluster itself for backups.
+Further it is often used by other applications such as logging and monitoring systems, or directly by the customer.
+
+Most platforms on which we run APPUiO Managed already provides some kind of S3 compatible storage and whenever possible AppCat directly interfaces with it to provide self-service object buckets.
+
+However, in some cases the underlying platform doesn't provide any object storage, and we don't have the option to use an off-site solution of another provider.
+In these cases we still need a self-contained storage solution that can provide S3 compatible storage and self-service object buckets on the platform.
```

Comment:
Like noted above: This is not only the case if there is no S3 compatible storage available, but also the case if there is S3 compatible storage available, but we don't have provider that allows us to dynamically provision buckets.